### DISCUSSION TITLE
这月饼有点......

### DISCUSSION THREAD

**Discussion by wangwenxi2580** [share121/inter-knot#5192] [其他]
![IMG_20240919_095822](https://github.com/user-attachments/assets/d23fafd3-e968-41f2-abac-870f0ac8d20e)

**Comment by Loxic123** [comment]
知らないでも、可愛いですね<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/32fc2d5e0929741455f506c6bd06e066_4709763683651054682.png" alt="米游姬 抛心心" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/32fc2d5e0929741455f506c6bd06e066_4709763683651054682.png" alt="米游姬 抛心心" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/32fc2d5e0929741455f506c6bd06e066_4709763683651054682.png" alt="米游姬 抛心心" width="75" >
